src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -113,8 +113,14 @@ def _polish_bad_errors(ex):
 
 # end of the new ones #
 # beginning of the old ones ###
+
+# in June, remove all olds stuff below and replace with one following line
+# cli_commands_removed('appservice web', "'az appservice web' is forever gone. You should use 'az webapp'")
+
 cli_command(__name__, 'appservice web create', 'azure.cli.command_modules.appservice.custom#create_webapp', exception_handler=ex_handler_factory())
-cli_command(__name__, 'appservice web list', 'azure.cli.command_modules.appservice.custom#list_webapp', table_transformer=transform_web_list_output)
+cli_command(__name__, 'appservice web list', 'azure.cli.command_modules.appservice.custom#list_webapp',
+            deprecating_info='webapp list',  # this will emit a warning and point to the new command
```

Comment:
I think deprecating info should be part of the help documentation (in `_help.py`). Here's the reason. In some cases, the deprecating information could be expanded to more than just redirect to another command. It may need a passage of text to explain the reason behind the deprecating, an example to highlight the migrating story or even a hyperlink to a more extensive detailed article. In that case, I think it is more manageable to treat it as help doc.